src/table/utils/model-store.ts

```diff
@@ -0,0 +1,17 @@
+const keyStore = (initialState: { [key: string]: EngineAPI.IField } = {}) => {
```

Comment:
I wonder if this can not be re-written to simply:
```js
const modelStore = new Map<string, EngineAPI.IField>();

export default modelStore;

```